src/OI.cpp

```diff
@@ -66,19 +66,23 @@ double OI::getDriverLeftXAxis() const {
 }
 
 double OI::getDriverLeftYAxis() const {
-  return driver->GetRawAxis(kLeftYAxis_ID);
+  return driver->GetRawAxis(kLeftYAxis_ID) * -1;
```

Comment:
They do not as since they are only needed if we wanted to return a positive throttle value as such I will remove them.